src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/ServiceController.php

```diff
@@ -143,7 +148,40 @@ public function create(Request $request): RedirectResponse|Response
             $this->logger->info(sprintf('Save new Service, service was created by: %s', '@todo'), (array) $command);
             try {
                 $this->commandBus->handle($command);
-                $this->addFlash('info', 'service.create.flash.success');
+
+                try {
+                    $service = $this->serviceService->getByLastInsertId();
+
+                    if ($service === null) {
+                        throw new InviteException('Could not find created Service.');
+                    }
+
+                    $message = $translator->trans('service.create.invitationMessage');
+
+                    if ($command->getTeamManagerEmail() === null || $service->getInviteRoleId() === null) {
+                        throw new InviteException('Could not create invite.');
+                    }
+
+                    $inviteCommand = new SendInviteCommand(
+                        $command->getTeamManagerEmail(),
+                        $message,
+                        'en',
```

Comment:
Good point.
I just checked with Bart to see what Invite accepts. 

Invite only accepts 'en'.

So currently, the SendInviteCommand only accepts 'en' as well in the validation.

So as far as I'm concerned, configuring the language in Invite is not a supported feature, as it might need to be mapped, or at least agreed upon the available values match between the systems.

The question then becomes, where to hardcode the 'en'. 

I left it here, so we have a half implemented configurable language :cat: 
I don't think it's a problem. Do you?

Reply:
discussed

Reply:
It would not be my preference to pollute a method with this hard coded value. But I can live with this!